lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1055,7 +1082,7 @@ func (w *WalletKit) prepareSweepParams(in *BumpFeeRequest,
 	// TODO(yy): upgrade `falafel` so we can make this field optional. Atm
 	// we cannot distinguish between user's not setting the field and
 	// setting it to 0.
-	if in.TargetConf != 0 {
+	if in.DeadlineDelta != 0 {
```

Comment:
Still using `in.TargetConf` in the line below this!

Reply:
ohh thank you !